modules/pbccs/main.nf

```diff
@@ -0,0 +1,55 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process PBCCS {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::pbccs=6.0.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/pbccs:6.0.0--h9ee0642_2"
+    } else {
+        container "quay.io/biocontainers/pbccs:6.0.0--h9ee0642_2"
+    }
+
+    input:
+    tuple val(meta), path(bam)
+    path pbi
```

Comment:
Sorry to contradict myself now, but I've seen a lot of modules having BAM + BAI associated with each other, so if you would rather have the pbi index with the BAM together I think this is OK.

Reply:
No problem. If it's allowed/tolerate, I prefer keep them together.